README.md

```diff
@@ -63,14 +63,12 @@ Add the following section to your package.json:
 ```json
 {
   "scripts": {
-    "one-version:check": "one-version check -p ${yarn | pnpm}"
+    "one-version:check": "one-version check"
```

Comment:
If we're going to make these changes to the readme, we should proabably plan to release right away otherwise this will be incorrect for the published version